src/Rule/AbstractCharCase.php

```diff
@@ -68,7 +68,7 @@ protected function strtoupper($str)
      * @return int
```

Comment:
```suggestion
     * @return int
```
The original declaration is wrong.